src/script/common/c_content.cpp

```diff
@@ -88,6 +88,14 @@ void read_item_definition(lua_State* L, int index,
 		def.tool_capabilities = new ToolCapabilities(
 				read_tool_capabilities(L, -1));
 	}
+	lua_getfield(L, index, "wear_color");
+	if (lua_istable(L, -1)) {
+		def.wear_bar_params = new WearBarParams(
+				read_wear_bar_params(L, -1));
```

Comment:
Making it in one line is fine.
```suggestion
		def.wear_bar_params = new WearBarParams(read_wear_bar_params(L, -1));
```